src/components/Contacts/ContactRow.tsx

```diff
@@ -31,22 +31,30 @@ const useStyles = makeStyles((theme: Theme) => ({
 }));
 interface Props {
   contact: ContactRowFragment;
+  onContactSelected: (contactId: string) => void;
 }
 
-export const ContactRow: React.FC<Props> = ({ contact }) => {
+export const ContactRow: React.FC<Props> = ({ contact, onContactSelected }) => {
   const classes = useStyles();
 
+  const onClick = () => {
+    onContactSelected(contact.id);
+  };
+
   return (
     <Box role="row" style={{ width: '100%' }}>
       <Box
+        role="rowButton"
         style={{
           height: '72px',
+          width: '100%',
           display: 'flex',
           flexDirection: 'row',
           justifyContent: 'space-evenly',
           alignItems: 'center',
           alignContent: 'center',
         }}
+        onClick={onClick}
       >
```

Comment:
What do you think about moving these styles out and using the ```styled``` API instead?
```
const  ContactRowButton = styled(Box)(({}) => ({
      height: '72px',
      width: '100%',
      display: 'flex',
      flexDirection: 'row',
      justifyContent: 'space-evenly',
      alignItems: 'center',
      alignContent: 'center',
}))

```

and then you would just have

```suggestion
<ContactRowButton
    role="rowButton"
    onClick={onClick}
>
```

I think the fewer inline styles the better as it makes the component tree a little bit more readable.